src/signals/settings/users/containers/Overview/hooks/useFetchUsers.js

```diff
@@ -25,16 +25,25 @@ const useFetchUsers = ({ page, pageSize } = {}) => {
     (async function fetchData() {
       setLoading(true);
 
+      const pageParams = [
+        page && `page=${page}`,
+        pageSize && `page_size=${pageSize}`,
+      ];
+
+      const filterParams = Object.entries(filters || {}).reduce(
+        (acc, [filter, value]) => value ? [...acc, `${filter}=${value}`] : acc,
```

Comment:
For readability, I would add a `filter` function, like so:
```jsx
const filterParams = Object.entries(filters || {})
  .filter(([, value]) => Boolean(value))
  .reduce((acc, [filter, value]) => [...acc, `${filter}=${value}`], []);
```
Just a suggestion, though